guides/mobile_app_guidance.md

```diff
@@ -0,0 +1,40 @@
+
+Building Cross-Platform Apps
+----------------------------
+
+It is possible that, having run through the hierarchy of Mobile Application development options as specified in the Mobile Application Standards the decision will have been reached to build a mobile app using a cross platform technology stack. These stacks are ways of using the same, or very close to the same, code-base but still build the app in ways that can allow it to run on multiple platforms and multiple devices. Otherwise a completely different code base, using a different language and development framework, is needed to build an app for each platform.
+
+There are many such cross platform technology stacks, but some are obscure or are nearing obsolescence. **The only cross platform stacks that should be considered for new projects, from the newest to the oldest, are Flutter, React Native or Xamarin.** All are backed by major vendors: Flutter by Google, React Native by Facebook and Xamarin by Microsoft, so all should have a reasonable shelf life, with Xamarin possibly the most "at risk" as it the oldest and has been overtaken by React Native and Flutter in terms of popularity (according to a 2018 Stack Overflow survey). Flutter requires developers to learn its own programming language, "Dart", though this is based on Java and C\# (as this is an open eco-system it is impossible to mandate an IDE, but Android Studio or Visual Studio Code are popular choices). React Native uses JavaScript as its programming language but also the whole React framework including a presentational technology called JSX, so is a relatively easy transition for React developers, but not so much for developers who know JavaScript but are not familiar with React (here again an IDE cannot be mandated, but Atom and Visual Studio Code are popular choices). Xamarin uses C\# so ought to be a reasonable match for .Net developers. Xamarin will require you to use full-blown Visual Studio for the IDE.
```

Comment:
Consider shortening and rewording as "We recommend using recent cross platform technology stacks such as Flutter, React Native or Xamarin", with links to their pages, and maybe which technologies they're most appropriate for.

Reply:
Quite a bit of research and some experimenting (with React Native) went into that and I think anyone would struggle to quickly find a shorter or more objective summary right now... especially as many sites are a bit "evangelical".